server/auth/auth.go

```diff
@@ -671,6 +671,27 @@ func (auth *AuthManager) Notify(acctID account.AccountID, msg *msgjson.Message,
 // Penalize signals that a user has broken a rule of community conduct, and that
 // their account should be penalized.
 func (auth *AuthManager) Penalize(user account.AccountID, rule account.Rule) error {
+	// Notify user of penalty.
+	details := "You may no longer trade. Leave your client running to complete outstanding trades."
+	penalty := &msgjson.Penalty{
+		Rule:     rule,
+		Time:     uint64(unixMsNow().Unix()),
+		Duration: uint64(time.Hour * 24 * 365 * 100), // 100 years
+		Details:  details,
```

Comment:
Having reflected on the penality notification more, we need to ensure the logging or display of the penalty notification includes the user id (the user is going to go to the server operator with what data?).  And on that node, it would really be desirable to have the details include the match id or other data that pertains to the violation.

Reply:
I think including a user ID in all signed messages would be a good idea, so the client both can be sure they were the intended target, and so they can later prove that specifically they were sent a message.

Reply:
Will do this in one more separate commit.

Reply:
Took the liberty of adding AccountID to Penalty. Hope that's ok.

I made the details argument just a string which is concatenated to the end of whatever details we send.